src/query/storages/system/src/user_functions_table.rs

```diff
@@ -140,6 +153,7 @@ impl UserFunctionsTable {
             TableField::new("arguments", TableDataType::Variant),
             TableField::new("language", TableDataType::String),
             TableField::new("definition", TableDataType::String),
+            TableField::new("created_on", TableDataType::Timestamp),
```

Comment:
How about TableDataType::Nullable(Box::from(TableDataType::Timestamp))? If can not get create_on return null?

Reply:
In meta default use UTC::Default is ok. This will consider forward compat, but in display , a function is created on 1970 is strange. BTW now only a very few users use udf, so display 1970 also ok.

Reply:
the created_on field in stage is referenced, so a default timestamp will be filled. Content that appears obviously "outdated" makes it easier for people to realize that they need to update udf.